### PR TITLE
Add _fragment route parameter as always available

### PR DESCRIPTION
**Pull request by cedricziel**
Let me know if this is really the right place. Do we need to check the projects' Symfony version? (If so: is there a tool already?)

Closes: #1071

### CODE REVIEW

File: src/fr/adrienbrault/idea/symfony2plugin/routing/RouteHelper.java [hunk 1]
```diff
@@ -85,6 +85,9 @@
             }
         }
 
+        // _fragment was introduced in Symfony 3.3 do we need a project version check?
```
Comment (Haehnchen) [inline src/fr/adrienbrault/idea/symfony2plugin/routing/RouteHelper.java]: as version check is just an if condition. whats about to add to it simply? `fr.adrienbrault.idea.symfony2plugin.util.SymfonyUtil#isVersionLessThenEquals`

Comment (cedricziel) [reply src/fr/adrienbrault/idea/symfony2plugin/routing/RouteHelper.java]: Great, didnt know the methods exists, will amend.

Comment (cedricziel) [reply src/fr/adrienbrault/idea/symfony2plugin/routing/RouteHelper.java]: Done.

**Comment by Haehnchen** [comment]
thx @cedricziel

**Comment by cedricziel** [comment]
Thank you for your efforts!

**Pull request merged**